client/options.go

```diff
@@ -309,6 +309,18 @@ type FlushOption func(*milvuspb.FlushRequest)
 
 type CreateDatabaseOption func(*milvuspb.CreateDatabaseRequest)
 
+func WithDatabaseProperty(key, value string) CreateDatabaseOption {
+	return func(req *milvuspb.CreateDatabaseRequest) {
+		if req.Properties == nil {
+			req.Properties = make([]*commonpb.KeyValuePair, 0)
+		}
+		req.Properties = append(req.Properties, &commonpb.KeyValuePair{
+			Key:   key,
+			Value: value,
+		})
```

Comment:
it is recommend to use a opt struct with map as inter-media so that duplicated key could be overwritten. plz check collection option implementation